### PR TITLE
Fixed Noelle level check

### PR DESCRIPTION
**Pull request by FireRainV**

### CODE REVIEW

**Comment by NyakoFox** [comment]
Has merge conflicts.

**Comment by FireRainV** [comment]
@NyakoFox I just fixed the conflict

**Comment by NyakoFox** [comment]
Still has merge conflicts, sorry.

File: data/party/noelle.lua [hunk 1]
```diff
@@ -103,18 +98,27 @@ function character:init()
...
-        return "LV"..self.level.." Ice Trancer\nReceives pain to\nbecome stronger."
-    elseif self:getFlag("iceshocks_used", 0) > 0 then
-        return "LV"..self.level.." Frostmancer\nFreezes the enemy."
-    else
-        return "LV1 "..self.title
-    end
+    
+    -- Display level (saved to the save file)
+    -- Default title / class (saved to the save file)
+    Game.stage.timer:everyInstant(1/30, function()
```
Comment (Dobby233Liu) [inline data/party/noelle.lua:107]: I kinda wonder if `0` works too. Still reads stupid though but IDK how to fix it

**Comment by NyakoFox** [comment]
Ideally it should use a function instead of a timer loop.

**Pull request closed without merging**